### PR TITLE
[READY]Makes ores stackable

### PR DESCRIPTION
**Pull request by silicons**
This should make ores stack as they will automatically get smelted to their smelted type. Stacks can be up to 50 each so this alone SHOULD reduce the amount of thrown objects. If there's anything else I can put in to make it less laggy, feel free to comment with it.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
does a new call and a qdel call every time an ore is touched really process faster than just throwing the ore again?

**Comment by kevinz000** [comment]
Uh, yes, because it effectively reduces the amount of thrown shit by 50 when it starts stacking which is the point where the server grinds to a halt.

File: code/modules/mining/ores_coins.dm [hunk 1]
```diff
@@ -7,6 +7,11 @@
 	var/points = 0 //How many points this ore gets you from the ore redemption machine
 	var/refined_type = null //What this ore defaults to being refined into
 
+/obj/item/weapon/ore/ex_act(severity)
+	if(refined_type)
+		new refined_type(get_turf(src.loc))
```
Comment (ChangelingRain) [inline code/modules/mining/ores_coins.dm]: downside; miners don't get the points
I suggest making ore a subtype of stacked stuff instead, so it stacks

Comment (kevinz000) [inline code/modules/mining/ores_coins.dm]: I mean, they didn't use any effort in it, but sure, I'l see if I can do that.

Comment (kevinz000) [inline code/modules/mining/ores_coins.dm]: Actually I just realized I needed to do it anyways, as sheets aren't as easily transported as ores.

**Comment by kevinz000** [comment]
It doesn't stack perfectly, but it is good enough that repeated bombings lag a lot, lot less.

**Comment by Shadowlight213** [comment]
\>Ready
\>Fails Travis
:ok_hand:

**Comment by kevinz000** [comment]
think i fixed it

**Comment by kevinz000** [comment]
fucking hell.

**Comment by kevinz000** [comment]
THERE WE GO.

**Comment by kevinz000** [comment]
and this is why i needed to bug test fuck dont merge this

**Comment by jughu** [comment]
When i was a QM with repulse and a miner was dropping ores at the orm (he had alot) i repulsed him for fun and nearly crashed the server with it then i told the gods WHEN WILL ORES BE STACKABLE? and now look at this!

**Comment by jughu** [comment]
Altougg kevinz how does this interract with mining bags can i carry 50 stacked ore of 50 or whaaat?

**Comment by kevinz000** [comment]
all gud

File: code/modules/mob/living/carbon/human/species_types/zombies.dm [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
+			C.unEquip(I, TRUE)
+
+
+// Your skin falls off
+/datum/species/krokodil_addict
+	name = "Human"
+	id = "goofzombies"
+	limbs_id = "zombie" //They look like zombies
+	sexes = 0
+	meat = /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/zombie
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/carbon/human/species_types/zombies.dm]: hi yes what does this have to do with ore?

File: code/modules/mob/living/carbon/human/species_types/synths.dm [hunk 3]
```diff
@@ -1,121 +1,121 @@
...
+	return list()
+
+
+/datum/species/synth/handle_speech(message, mob/living/carbon/human/H)
+	if(H.health > disguise_fail_health)
+		if(fake_species)
+			return fake_species.handle_speech(message,H)
+		else
+			return ..()
+	else
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/carbon/human/species_types/synths.dm]: uhh?

**Comment by ExcessiveUseOfCobblestone** [comment]
this pr is ill

**Comment by kevinz000** [comment]
WHAT THE FUCK AAA

**Comment by jughu** [comment]
This is just gonne let us enable to carry 2500 ores in mining bags im sure of it!

**Comment by kevinz000** [comment]
@jughu no bags properly calculate amounts.

**Comment by Jordie0608** [comment]
This PR is diagnosed with a serious case of the line endings; the condition is fatal if not treated soon.

**Comment by kevinz000** [comment]
mso's thing fixed server crashing by bombing so i'm not going to bother fixing a system that already works
